dbt_project.yml

```diff
@@ -29,20 +29,21 @@ vars:
     # snowplow__database: Only set if not using target.database for Snowplow events data -- WILL BE IGNORED FOR DATABRICKS
     # snowplow__events_table: "events" # Only set if not using 'events' table for Snowplow events data
     snowplow__events: "{{ source('atomic', 'events') }}"
-    snowplow__session_context: "{{ source('atomic','com_snowplowanalytics_snowplow_client_session_1') }}"
-    snowplow__mobile_context: "{{ source('atomic','com_snowplowanalytics_snowplow_mobile_context_1') }}"
-    snowplow__geolocation_context: "{{ source('atomic','com_snowplowanalytics_snowplow_geolocation_context_1') }}"
-    snowplow__application_context: "{{ source('atomic','com_snowplowanalytics_mobile_application_1') }}"
-    snowplow__screen_context: "{{ source('atomic','com_snowplowanalytics_mobile_screen_1') }}"
-    snowplow__app_errors_table: "{{ source('atomic','com_snowplowanalytics_snowplow_application_error_1') }}"
-    snowplow__screen_view_events: "{{ source('atomic','com_snowplowanalytics_mobile_screen_view_1') }}"
+    snowplow__session_context: "com_snowplowanalytics_snowplow_client_session_1"
+    snowplow__mobile_context: "com_snowplowanalytics_snowplow_mobile_context_1"
+    snowplow__geolocation_context: "com_snowplowanalytics_snowplow_geolocation_context_1"
+    snowplow__application_context: "com_snowplowanalytics_mobile_application_1"
+    snowplow__screen_context: "com_snowplowanalytics_mobile_screen_1"
+    snowplow__app_errors_context: "com_snowplowanalytics_snowplow_application_error_1"
```

Comment:
Why the change here? If it is an sde it is better to rename it to `_events`. FYI I started to follow this convention in the unified model: 
<img width="644" alt="image" src="https://github.com/snowplow/dbt-snowplow-mobile/assets/95634439/b3ec01d9-e33a-4f6c-a663-394aed07df3b">

Reply:
Yep, you're right!